Formula/asimov.rb

```diff
@@ -33,7 +33,7 @@ def plist
   end
 
   test do
-    assert_match %r{Finding node_modules/ directories with corresponding ../package.json files...},
+    assert_match /Finding dependency directories with corresponding definition files…/,
```

Comment:
No need for an actual regex definition here
```suggestion
    assert_match "Finding dependency directories with corresponding definition files…",
```

Reply:
Excellent point; I got so caught up in the standards telling me not to use the regex literal that I didn't think "hey, this doesn't really need a regex at this point anymore."